pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/multifile/ApexMultifileAnalysisTest.java

```diff
@@ -63,6 +63,8 @@ void testWellFormedSfdxProjectJsonProducesFunctionalAnalysis() throws Exception
 
             assertFalse(analysisInstance.isFailed());
         });
+
+        // TODO: This is failing due to ANTLR versions, 4.9.1 vs 4.8, expect to resolve with apex-dev-tools switch
```

Comment:
ok, I understand now, why this test fails only now: Before we initialized Org for every run regardless whether it was a sdfx project. The messages were printed, but for another test already. The messages are printed only once (static initializer in ApexParser).

Now we initialize Org only for sfdx projects and this test case is the first time this is done and therefore the messages are printed there.

I'll keep the TODO but remove the messages from the log, so that the test works.